content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -174,6 +174,48 @@ spec:
 
 However, the particular path specified in the custom recycler Pod template in the `volumes` part is replaced with the particular path of the volume that is being recycled.
 
+### Pre-binding
+
+The control plane can [bind PVCs to matching PVs](#binding) in the
+cluster. However, if you want a PVC to bind to a specific PV, you need to
+pre-bind them.
+
+You can specify the PV in your PVC, then the PVC will be able to bind to this
+specific PV. If the PV exists and is not reserved for other PVCs (has PVC
+specified in `claimRef` field), the PV and PVC will be bound regardless of
+whether the PV satisfies some matching criteria, e.g. node affinity. Storage
+class, access modes, and requested storage size are still checked.
+
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: foo-pvc
+  namespace: foo
+spec:
+  volumeName: foo-pv
+  ...
+```
+
+Note that this method does not guarantee the PV right. If the PV you specified
```

Comment:
Hi @cofyc . Several doc-related questions and suggestions.
nit: You could spell out the terms instead of only using the abbreviations. I find it hard to read about `PV` and `PVC`.
Question: What method? A method described in the previous paragraph? The `pre-binding` method?
nit: Wording, the phrase, "This method does not guarantee any [what?] privileges to the Persistent volume? 
nit: What do you think about adding this information using a `note` shortcode?

nit: If your specified Persistent volume can be used by other Persistent Volume Claims, you need to reserve ...

Reply:
```suggestion
Note that this method does not guarantee the persistent volume right. If the persistent volume you specified
```

Reply:
> nit: You could spell out the terms instead of only using the abbreviations. I find it hard to read about PV and PVC.

done, thanks @celestehorgan ! but the abbreviations PV & PVC are widely used in this doc (https://kubernetes.io/docs/concepts/storage/persistent-volumes/). Is it necessary?

> Question: What method? A method described in the previous paragraph? The pre-binding method?

the method described in the previous paragraph

> nit: Wording, the phrase, "This method does not guarantee any binding privileges to the Persistent volume?
> nit: What do you think about adding this information using a note shortcode?
> nit: If your specified Persistent volume can be used by other Persistent Volume Claims, you need to reserve ...

this introduces the reserve approach, I think it's better not to add this information using a note shortcode.

I've updated the PR and fixed the wording too. Thanks! PTAL.